cli/azd/cmd/show.go

```diff
@@ -165,7 +178,82 @@ func (s *showAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 		}
 	}
 
-	return nil, s.formatter.Format(res, s.writer, nil)
+	if s.formatter.Kind() == output.JsonFormat {
+		return nil, s.formatter.Format(res, s.writer, nil)
+	}
+
+	appEnvironments, err := s.envManager.List(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	uxEnvironments := make([]*ux.ShowEnvironment, len(appEnvironments))
+	for index, environment := range appEnvironments {
+		uxEnvironments[index] = &ux.ShowEnvironment{
+			Name:      environment.Name,
+			IsCurrent: environment.Name == environmentName,
+			IsRemote:  !environment.HasLocal && environment.HasRemote,
```

Comment:
Data model here feels a little off here. An environment "has remote" and "has local" components, but we merge it to mean "IsRemote". I see that this ends up tagging the environment as "<env> (Remote)" which is fine but would be confusing if we ever presented "<env> with IsRemote: true/false".